Add new column to output of show transaction clause (#1501)
neo4j/docs-cypher: modules/ROOT/pages/clauses/transaction-clauses.adoc

a| The initialization stacktrace for this transaction, or an empty `STRING` if unavailable.
m| STRING

m| currentQueryProgress
a| A map containing real time metrics of data modifications made by the query currently executing in this transaction, or null if the current transaction is an inner query.

> The metrics of data modifications made by the inner queries will be aggregated in their outer query on commit. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.03]

Reviewer: ```suggestion
The metrics of data modifications made by the inner queries are aggregated in their outer query on commit. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.03]
```

> The metrics of data modifications made by the inner queries will be aggregated in their outer query on commit. label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2026.03]

Reviewer: what does "on commit" mean?

Reviewer 2: A query can have several inner queries, then the stats from the inner queries will be aggregated and shown in the outer query. However this will not happen until the inner query is finished and committed. So this data will not be shown in "real time", but will be reported as inner transactions commit.

m| MAP

|===

.Contents of the currentQueryProgress map

> * `nodesCreated` Total number of nodes created by the query currently executing in this transaction.

Reviewer 2: > by the query currently executing in this transaction.

In the `currentQueryProgress` row in the table above we already state that the metrics is for the "query currently executing in this transaction". Do we want to remove this part of the sentence for all the items in the content list, since it becomes quite repetitive, perhaps?

Author: Makes sense to me

* `nodesDeleted` Total number of nodes deleted.
* `relationshipsCreated` Total number of relationships created.
* `relationshipsDeleted` Total number of relationships deleted.
* `propertiesSet` Total number of properties set.
* `labelsAdded` Total numbers of labels added.
* `labelsRemoved` Total numbers of labels removed.

> * `fileLinesRead` Total number of CVS file lines read.

Author: ```suggestion
* `fileLinesRead` Total number of CSV file lines read.
```

typo
